thirdparty/runtime_environments.go

```diff
@@ -69,14 +71,22 @@ func (c *RuntimeEnvironmentsClient) GetImageNames(ctx context.Context) ([]string
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
 		msg, _ := io.ReadAll(resp.Body)
-		return nil, errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
+		apiErr := errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
```

Comment:
honestly i don't think there's a benefit to wrapping msg here, and having things be separate fields when possible is always helpful, so I'd separate resp.Status into a "status" field, and just wrapping "msg" directly with errors.New. (This is because you can build the splunk alert on message, so you may as well keep the error simple to what you need; the wrapped message above is more user-friendly than we need)

Reply:
I'll just keep `status_code`! (unless you think I should also add `status`)